lib/datasource/local/settings_storage.dart

```diff
@@ -92,6 +102,13 @@ class _SettingsStorage {
     }
   }
 
+  set privateKey(String? value) {
```

Comment:
Why is there both a private key and a list of private keys?

Reply:
Because I had it in mind when an account was changed, in addition to being saved in the lists (accounts and provateKeys) I thought only to update the current fields of accountName and privateKey